MobileFilter/sections/specific_web.txt

```diff
@@ -4257,6 +4257,13 @@ popularmechanics.com#?#section > div[class*=" "]:has(> div[class]:first-child +
 veranda.com,wetter.com#?#div[class^="sc-"]:has(> div.ad-container)
 wetter.com#?#.som-wrapper:has(> div[data-slotname])
 wetter.com#?#.clickperformance-wrapper:has(> .ad-flex)
+wetter.com#?#.mbanner-fixed:remove()
+wetter.com#$#.mbanner-is-fixed { top: 0px !important; padding-top: 0px !important; }
+wetter.com#$##main-header-fixable { top: 0px !important; }
+wetter.com#?#div.ad-container:remove()
+wetter.com#?#div:has(> div#div-gpt-ad-wetter_direct_toolbar):remove()
```

Comment:
```
wetter.com#?#div:has(> div#div-gpt-ad-wetter_direct_toolbar)
```

Is `:remove()` required? `wetter.com##div:has(> div#div-gpt-ad-wetter_direct_toolbar)` works fine on my device, search on the main page.

Reply:
Same as https://github.com/AdguardTeam/AdguardFilters/pull/180846#discussion_r1627921752